apps/total-typescript/src/components/exercise-overlay.tsx

```diff
@@ -244,6 +263,26 @@ const FinishedOverlay = () => {
     addProgressMutation.mutate({lessonSlug: lesson.slug})
   }, [])
 
+  const handlePlayFromBeginning = () => {
+    router
+      .push({
+        pathname: section
+          ? `/${path}/[module]/[section]/[exercise]`
+          : `/${path}/[module]/[exercise]`,
```

Comment:
this isn't nice looking and we could probably use something similar to [getResourcePaths](https://github.com/skillrecordings/products/blob/main/apps/testingaccessibility/src/utils/get-resource-paths.ts) util